src/content/en/updates/2017/12/modulepreload.md

```diff
@@ -0,0 +1,138 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: &lt;link rel="modulepreload"&gt; offers a way of declaratively loading ES2015 modules ahead of time.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_tags: performance #}
+{# wf_blink_components: Blink>Loader #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
```

Comment:
just checking, is https://developers.google.com/web/updates/images/generic/timeline.png the image you want?

Reply:
Yes, couldn't really find anything more suitable :)